lib/swarm/distribution/static_quorum_ring.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Swarm.Distribution.StaticQuorumRing do
+  @moduledoc """
+  A quorum is the minimum number of nodes that a distributed cluster has to obtain in order to be
+  allowed to perform an operation. This can be used to enforce consistent operation in a distributed system.
+
+  ## Quorum size
+
+  You must configure the quorum size using the `:static_quorum_size` setting:
+
+      config :swarm,
+        static_quorum_size: 5
+
+  It defines the minimum number of nodes that must be connected in the cluster to allow process
+  registration and distribution.
+
+  If there are fewer nodes currently available than the quorum size, any calls to
```

Comment:
It is for sure :). The only caveat to the above is that the configuration needs to be present when the VM boots, so running under mix, you need to pass `--erl "-config path/to/sys.config"` and convert the configuration I mentioned to Erlang terms, e.g.:

```erlang
[{kernel, [{sync_nodes_mandatory, ['node1@192.168.1.1', ...]},
                 {sync_nodes_timeout, 60000}]}].
```

Using the Mix config files works for releases though.